src/VirtualClient/VirtualClient.Core/ProfileExecutor.cs

```diff
@@ -708,6 +707,11 @@ private static void OutputComponentStart(string componentType, VirtualClientComp
                             runtimeComponent.Metadata.AddRange(this.Metadata, true);
                         }
 
+                        if (this.Profile.Parameters?.Any() == true)
+                        {
+                            runtimeComponent.Parameters.AddRange(this.Profile.Parameters, true);
```

Comment:
Provide visibility of all commandline parameters to profile

Reply:
Is there a case the current implementation doesn't work? I'm worried that we might add parameters to say SPECjbb, to another workloads or monitor. Functionally it's fine but it bloats the telemetry

Reply:
From my understanding, the Profile.Parameters has the global parameters that are provided either through Profile parameters or from CommandLine. While, the vcComponent.Parameters has action/monitor level parameters. I modified the AddRange to not replace the keys in dictionary if already present, so any parameter defined in action would not be overwritten here by global parameter here.